Kubernetes/linux/manifest/generate.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/python2
+import sys
+import os.path
+
+# Inputs:
+#
+#   - Master IP, the external IP address of the masas inter
+#   - Cluster CIDR, likely a /16
+#
+# Outputs:
+#   - 5 manifest files:
+#       kube-etcd.yaml
+#       kube-scheduler.yaml
+#       kube-apiserver.yaml
+#       kube-addon-manager.yaml
+#       kube-control-manager.yaml
+
+MANIFEST_TEMPLATES = {
+    "kube-addon-manager.yaml": """apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-addon-manager
+  namespace: kube-system
+  version: v1
+spec:
+  hostNetwork: true
+  containers:
+  - name: kube-addon-manager
+    image: gcr.io/google_containers/kube-addon-manager-amd64:v6.4-beta.2
+    resources:
+      requests:
+        cpu: 5m
+        memory: 50Mi
+    volumeMounts:
+    - name: addons
+      mountPath: "/etc/kubernetes/addons"
+      readOnly: true
+  volumes:
+  - name: addons
+    hostPath:
+      path: "$HOME/kube/addons"
+""",
+    "kube-apiserver.yaml": """
+    apiVersion: "v1"

```

Comment:
The whitespace here before "apiVersion" is a problem. It seemed to cause the resultant kube-apiserver.yaml in my deployment to be invalid and therefore apiserver failed to come up as a result. I manually removed the whitespace from the yaml and then things started working.